### ISSUE TITLE
[BUG] Valid Auth Token not working

### ISSUE THREAD

**Issue by jramay** [phamleduy04/texas-dps-scheduler#176]
**Before submitting please make sure:**
- [Yes] I have checked all of the other issues and there is no duplication of this issue.

Environment:
- What environment are you using?` Docker`
- What version of the app are you running? `v4.3.0`

**Describe the bug**
Followed the [steps](https://github.com/phamleduy04/texas-dps-scheduler/wiki/How-to-get-Captcha-Token) to get Captcha Token and updated in it `config.xml` but got following failure immediately without single successful execution of API.

```
texas-dps-scheduler  | [08/22/2024 11:42:22] Auth token expired! Try to get new token...
texas-dps-scheduler  | [08/22/2024 11:42:22] Captcha token found: 103AFcWeA6-H7ulWzSAAQydoEM435MhDZ37j4KUSGehAZbUD1K7zBjUUzrLKB1GaxU7n6wNahIECbg4Gnf2eqssoXI6Y7NWH8CzsTsJvq4DidWJwwsDHKr84sG_GkcjD_V5-3lBenVXMZ_YtzuVjNLVkd9Gk77AxxQ0wg6umSJGkpOIS2o9HNhATPfTB9ez1uRSM9r9DxN95E2qxlXCiph24wq1W3_icl0i6fW6Ev_7uM570KIh4PuInMTZKZ25Gqq3ndcK8D4yyUxn0MrXUAQ6y2YO3d7sGYRXvEKvpNxed5niVBAAzkzrd_kUpEhAOnj-o6cV1LvK1vpfc-vvLgpoKKZfGzntex2Mqffud9R0iWlfi3osgGtqHBC-GvLw6w0yb_5TpOpiF4RyLumFvUVRweFOlJK5RFDLAPvPcj4O4F-JFwi9pj2SFgaFZFCYlJ3wEaPfU7TD_uor4qYJ5OsnWF-_4heYx-b5-d5vHAApKCkEkRl4TciRpaK-FQDlKrduROmVuvg6oj57AdCvTqUswtrUhbWElO5SNMcvOq_1ZFO4_ixbha7XFfZ_4wmbfhlB5dxEJXDh2QUlXlWjHNC13U-RsqGm7oJSvjaRkWVlznvffiTnN4uEAHzqeREjA_hFpeH8zxyn3d0n9grchCOjT0SyDnYbdr-YCsheUNB3HNvKTP097Z26nJroC1n1XkOPRD3h1MkT6ki9HvZyTTNu0QbjiKgZPkxetZUKFvPshfVpPxXHP06HjkGAR4L_XJuQWH2W1yIV4warB2ipIbNOzBgLoZ3pNL5vdwj6r2kOe20CbM7m1RsjH279jwfbC3PDFsfpc_GqXeFjZxzTSQYISTx0E7-9JiE37NIXMU2sDvL6TJ-qF3P94uurs71lc83U4F7knJL9rixSQwWbejWi04MP0Gn6Py_LrcKsIMxG6Zd6XFIR_qU27GNN_kHBRHiIofOMoLh2mMEGP5nVihu_V59fguO_0MQwE0jF79SUYNZnZaK00ZUl71zxAYASQAk38oMSO-fq6yuwhbWfdlGSOZCD0Cy2iL3HA9s9mXlhPDb6U_u7b3Av4FFvPCytdYlxKcjW9rtA9Y83HhuOjzMZy5J1K0X4PBwmI62M2XotLK4-kVsC-2FX0uveEXUc1xeZEFjOt4WecQd8hOVpI7oQSC5_ytUMGTdORme-QlE9ojyl4F-SMVyNrRwFW3sIfJ2-Z_-XfTHOObMkDXSarO57_kaQzx0yGsS_OAZptFe5x0qoFm0ArMAHvFwO4ZPRIJfhq6qBCF_Pjc5k_-UFpFb_TwASzDshfz1WxAd56Z8mrlNvu0ZpD686ZK7rnAtu4GiPeUl_sUAA9bwa07PapEDjvRj9MugWE-q4KMT4KJHaTbP2w1CMgDXpK5CTKo0JO80LqPWbN_fS1QZL42VMyg2LnKDZNbGNrOob6dMr2g91V1dFbGElTiKH4Z8ZqiylwLySMf0SXfWvMJ2zmj73nMsNINPDLDnN1jlZRSka9xzdKWD9URjpopAYFQAzEwyLsFWRf0LW_Zczs65W0v8c8BWyr16sSV3vDr2MN9ttehU23JmBW_-NLZszEYa_1_owABPmjhtmkaAxBxwKWy1JwVFBClnJMyOntXSXZTIRN9EOZ9EI7zCllpQ2V8W1sjQ9-X26fUowleeeCwJHXdEQiX-P-zveDaImsoKhIwG7_tFr5HiGiQLm18Nm2EQXrmJrndXeQqcrJa2MbS268AcgqXP6TQXzq3e6ppg_8WcTTMfInKVhnCk63LzC-hpaKwX_hAf8SOuK4c81uRLpj21GL2jKQCnOSA0u048pHocJemx7d0Zx85LZGm9QvjVkpTQYyfbTTIDa0EGFccsrN8Feies0UhKn_0t1mf1teg8Bdjvl86EM1hFWGdZT4-0Lx-XCNPG6wXZMEYUkcyXFQguF2eKo5wS2MOZBuFheUpnhCtO3FGI3SDZEr2sbxtGNZgX5Ek3gLyJuPts4mqK36egeyf_L5fe9mSAhhhj67Foo7jOGXjS9CJ3XT4lZg2hwjOA_j9Iw6sDoNRq5-XqwI4Jh1Ubb6Hb4Y6MGNoqgh5eZKu_5t9t5gkEtg81UTs4oHOHV5Y9GaVp_SniNzzZsfMbwBBmMhg5sqEvBrz6ofS4_RNFdNlIxvaP1v5cRWekdR1SHaJvhxoK3bDWEtsG2
texas-dps-scheduler  | [08/22/2024 11:42:22] WARNING -> Got 401 status code
texas-dps-scheduler  | [08/22/2024 11:42:22] WARNING -> Got 403 status code
```

**To Reproduce**
Steps to reproduce the behaviour:
1. Follow the [steps](https://github.com/phamleduy04/texas-dps-scheduler/wiki/How-to-get-Captcha-Token) to get Captcha Token and update in it `config.xml`
2. Run the docker container.
3. See the error in the logs.

**Expected behavior**
After providing valid Captcha Token the script should run without any 40x Status Codes.

**Screenshots**
Not Applicable.

**Additional context**
Not Applicable.

**Comment by phamleduy04** [comment]
duplicate of #175

**Comment by phamleduy04** [comment]
fixed in v5.0.0
